src/platformio.ini

```diff
@@ -189,6 +189,58 @@ lib_deps =
 	Wire
 monitor_speed = 420000
 
+[env:Frsky_RX_R900MINI_via_STLINK]
+platform = ststm32@8.0.0
+framework = arduino
+board = R9MM
+build_unflags = -Os
+build_flags =
+	-D TARGET_R9M_RX
+	-D TARGET_R900MINI_RX
+	-D TARGET_100mW_MODULE
+	-D PLATFORM_STM32
+	-D HSE_VALUE=24000000U
+	-O2
+	-DVECT_TAB_OFFSET=0x08008000U
+board_build.ldscript = variants/R9MM/R9MM_ldscript.ld
```

Comment:
not sure if we need to change this, I am not familiar with those scripts

Reply:
No need to duplicate it just for different naming. These linker files should be merged to one later.

Reply:
yeah, I saw on some targets they use inheritance from others, so probably it will be reworked at some point